shardingsphere-test/shardingsphere-test-common/src/main/java/org/apache/shardingsphere/test/runner/ShardingSphereParallelTestParameterized.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.test.runner;
+
+import org.apache.shardingsphere.test.runner.parallel.DefaultParallelRunnerExecutorFactory;
+import org.apache.shardingsphere.test.runner.parallel.ParallelRunnerScheduler;
+import org.apache.shardingsphere.test.runner.parallel.annotaion.ParallelLevel;
+import org.apache.shardingsphere.test.runner.parallel.annotaion.ParallelRuntimeStrategy;
+import org.junit.runners.Parameterized;
+
+/**
+ * ShardingSphere integration test parameterized.
+ */
+public final class ShardingSphereParallelTestParameterized extends Parameterized {
```

Comment:
The class `ShardingSphereIntegrationTestParameterized`  had still existed.
How about consider about reuse it?

Reply:
yes, this pr is based on the implement of framework of 
 ShardingSphereIntegrationTestParameterized.  And the reason refactoring  to move the implements to test-common package is that unit test cases could use this framework for parallel execution which  just need to depend on test-common. And from my first thought is that ShardingSphereIntegrationTestParameterized is for integration test; while most of unit test cases don't want to depend on integration-test-suite package. and integration-test-suite  could rely on test-common package. and I can't just move the framework implement from integration-test-suite package to test-common package, because the implement use some classes in integration modules and I don't want test-common package depend on it.  so the basic idea of this refactor is that make an default parallel execution framework in test-common which follow the implement in integration-test-suite other than the part related with integration test, then other modules could use this framework in test-common for parallel testing . and the origin implement in  integration-test-suite extend the framework in test-common  for specific integration test cases.  
thanks.